src/main/java/com/minecolonies/coremod/client/gui/WindowRequestDetail.java

```diff
@@ -206,6 +232,48 @@ public void onOpened()
         box.setSize(box.getWidth(), y);
     }
 
+    /**
+     * Checks if the request is fulfillable
+     */
+    public Boolean fulfillable(){
```

Comment:
formatting is off here, did you import the formatting schema from the /documentation/ folder?